src/components/Auth.js

```diff
@@ -66,6 +82,35 @@ class Auth extends Component {
     return false
   }
 
+  async onCreate (e) {
+    e.preventDefault()
+
+    // setting the model dialog state to be true 
+    this.setState({
+      "modelOpen": true
```

Comment:
Here you don't need to use quotes, since JS will already consider it as a String. Also, I believe it is "modal" instead of "model", but the modal component from material-ui is called "dialog". We can use "dialogOpen" or even "signUpOpen" to clearly state the purpose of this state. The function names should also be changed.

An example about object keys on Javascript: 
```javascript
var state = { opened: true }
console.log(state.opened) // true

var state = { 'opened': true }
console.log(state.opened) // true (it is the same thing as the example above)

var myStateName = 'opened'
var state = { myStateName: true }
console.log(state.opened) // undefined!
console.log(state.myStateName) // true!

var state = { [myStateName]: true }
console.log(state.opened) // true
console.log(state.myStateName) // undefined
```

Reply:
Thanks for sharing this. I will make the changes.